src/components/EditCoursePage/CollapsibleCourseRun.test.jsx

```diff
@@ -15,8 +14,7 @@ import {
 import { courseSubmitRun } from '../../data/actions/courseSubmitInfo';
 import CollapsibleCourseRun from './CollapsibleCourseRun';
 
-const mockStore = configureStore();
-const store = mockStore({});
+import store from '../../data/store';
```

Comment:
Why this import instead of mock? Is this an RTL recommendation?

Reply:
mock? Isn't it import? The only store related mock in the file is store.dispatch.

Reply:
I was referring to the need to mock the `dispatch` on the store that is being used by the component, which is the imported one, and not the earlier "mocked" one.